cli.go

```diff
@@ -967,21 +978,23 @@ func main() {
 						}
 					}
 
-					projectListKeys := c.StringSlice("list-key")
-					projectListNames := c.StringSlice("list")
-
-					if len(projectListNames) > 0 {
-						// Add project lists by name:
+					if len(projectListNames) > 0 || doAllLists {
 						lists, err := client.ListProjectSelections()
 						if err != nil {
 							panic(err)
 						}
 
-						for _, name := range projectListNames {
```

Comment:
This range loop is needed.

Reply:
Sometimes I doubt if there's an evil mastermind controlling me while committing... 😂 will get that loop back tomorrow 👍